### DISCUSSION TITLE
Fowl Play compatibility datapack template

### DISCUSSION THREAD

**Discussion by aqariio** [aqariio/Fowl-Play#28] [Announcements]
This is the template for a datapack that modifies the biomes where the birds spawn. Inside you will find files named "spawns_<mob name>.json". To add spawning biomes to a specific mob, you would add the biome id or a biome tag inside the file. Remember to add a # before biome tags!

If you think I should add built-in compatibility to any biome mods, or that I've missed some biomes, please feel free to make an issue listing the biomes!

[fowlplay-compat-template.zip](https://github.com/user-attachments/files/18052758/fowlplay-compat-template.zip)

**Comment by muhin976-beep** [comment]
Привет, заполняю по шаблону:
{
  "values": [
    { "id": "#minecraft:forest", "required": false },
    { "id": "#minecraft:plains", "required": false },
    { "id": "#biomesoplenty:seasonal_forest", "required": false },
    { "id": "#biomesoplenty:woodland", "required": false },
    { "id": "#biomesoplenty:orchard", "required": false },
    { "id": "#biomesoplenty:old_growth_woodland", "required": false }
  ]
}
но почему то мобы не появляются в данных локациях - может чего то не хватает?

**Reply by aqariio** [reply]
Can you send your datapack so that I can take a look?

**Reply by muhin976-beep** [reply]
[fowlplay-1.1.0-beta.4-1.21.1-neoforge.zip](https://github.com/user-attachments/files/23623646/fowlplay-1.1.0-beta.4-1.21.1-neoforge.zip)

**Reply by muhin976-beep** [reply]
bro, I also noticed that I don't see any /summon entities

**Reply by muhin976-beep** [reply]
and forget about it, I think I have a version conflict and it doesn't load on Neferod 1.21.1..... I just opened the console with the list of mods, and it's not there, even though it's in the mods folder
